javascript/javascript1/week4/js-week4.js

```diff
@@ -0,0 +1,111 @@
+let userName="";
+let toDoList=[];
+function getReply(command){
+    command=command.toLowerCase();
```

Comment:
Try to leave some air 😮‍💨(spaces) between names and equal signs for the sake of the code's clarity. 😊